index.bs

```diff
@@ -34,23 +34,23 @@ spec: RFC6455; urlPrefix: https://tools.ietf.org/html/rfc6455
     text: Fail the WebSocket Connection; url: section-7.1.7
     text: Status Codes; url: section-7.4
     text: Handling Errors in UTF-8-Encoded Data; url: section-8.1
-spec: RFC8610; urlPrefix: https://tools.ietf.org/html/rfc8610
+spec: RFC8610; urlPrefix: https://datatracker.ietf.org/doc/html/rfc6455#section-3
   type: dfn
     text: match a CDDL specification; url: appendix-C
 spec: RFC6265; urlPrefix: https://httpwg.org/specs/rfc6265.html
   type: dfn
-    text: Cookie; url: section-5.3
-    text: Cookie store; url: section-5.3
-spec: RFC6265bis; urlPrefix: https://tools.ietf.org/html/draft-ietf-httpbis-rfc6265bis-05
+    text: Cookie; url: storage-model
```

Comment:
It feels like this was always linking to the wrong thing. Seems like maybe it should just link to `#intro` for the definition of a Cookie.

Reply:
I updated the link to target the whole RFC (no fragment), which seems good enough?

Reply:
I think the "Cookie store" link should revert to targeting section 5.3 which describes how a client stores cookies.

Reply:
But a link to the whole document is fine for `Cookie`.